### PR TITLE
fix(README): Add a README.md and how to release a new version of this repository

### PR DESCRIPTION
**Pull request by brandon-welsch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Docker Cleaner v0.4.0
+
+Small ruby script to remove old containers and old images.
```
Comment (EtienneM) [inline README.md:3]: Can you specify what is "old", and where does it remove the containers and images from? Does the script remove them from builders, apps node, etc?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Release a New Version
+
+Bump new version number in:
+- README.md
+- lib/docker_cleaner/version.rb
+
+Commit, tag and create a new release:
+```shell
+git add README.md
```
Comment (EtienneM) [inline README.md:13]:
```suggestion
git add README.md lib/docker_cleaner/version.rb
```

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Soulou) [reply README.md:3]: docker-cleaner is agnostic from Scalingo @EtienneM it removes them from any host it's installed on and run manually or as a cron task

File: README.md (hunk 2 above)
Comment (brandon-welsch) [reply README.md:13]: Great catch

File: README.md (hunk 1 above)
Comment (brandon-welsch) [reply README.md:3]: @EtienneM Is that OK now?

Comment (EtienneM) [reply README.md:3]: Oh I didn't see it's a public project

**Review by EtienneM — APPROVED** [review]

**Review by Soulou — APPROVED** [review]

**Pull request merged**